boards.txt

```diff
@@ -17,6 +17,7 @@ menu.EventsCore=Events Run On
 menu.MemoryType=Memory Type
 menu.EraseFlash=Erase All Flash Before Sketch Upload
 menu.JTAGAdapter=JTAG Adapter
+menu.Zigbee=Zigbee Mode
```

Comment:
please rename the menu to `ZigbeeMode`. This is a bit too generic

Reply:
Done.